infrastructure/prod.tfvars

```diff
@@ -43,6 +43,8 @@ worker_ecs_log_group_name         = "crossfeed-prod-worker"
 worker_ecs_role_name              = "crossfeed-prod-worker"
 logging_bucket_name               = "cisa-crossfeed-prod-logging"
 export_bucket_name                = "cisa-crossfeed-prod-exports"
+pe_reports_bucket_name            = "cisa-crossfeed-pe-reports"
```

Comment:
I think we should give them different names in staging and prod. otherwise they might overwrite each other...